setup.py

```diff
@@ -2,17 +2,35 @@
 from setuptools import setup, find_packages
 
 
+VERSION = '0.3.0'
+
+
 setup(
   name='AeroGear build cli for Digger',
-  version='0.3.0',
+  version=VERSION,
   description='AeroGear build cli for Digger',
   author='AeroGear',
+  author_email = 'aerogear-dev@lists.jboss.org',
```

Comment:
Yes, but that e-mail is for contacting the author, PyPIs username and password is used in another file which is not in this PR.